index.js

```diff
@@ -126,6 +126,11 @@ export default class ShockWallet extends React.Component {
   }
 }
 
+/**
+ * @param {string} path
+ */
+const sanitizePath = path => path.replace(/\//g, '_')
```

Comment:
Please add a comment describing what the regex does.

Reply:
Same as above